{{cookiecutter.repostory_name}}/nginx/templates/default.conf.template

```diff
@@ -98,6 +98,11 @@ server {
             return 425;
         }
         {%- endif %}
+
+        {% if cookiecutter.nginx_filter_bad_host_headers == 'y' -%}
+        include /etc/nginx/config_helpers/ignore_bad_host.conf;
+        {%- endif %}
+
```

Comment:
seeing how we have
>     server_name               ${NGINX_HOST} {% if cookiecutter.nginx_redirect_www_prefix == 'n' %}www.${NGINX_HOST}{% endif %};

on top, can you elaborate how does this help?

Reply:
This is a mistake, I put it in the wrong block